patches/net/minecraft/world/item/ItemStack.java.patch

```diff
@@ -158,15 +158,23 @@
      public ItemEnchantments getEnchantments() {
          return this.getOrDefault(DataComponents.ENCHANTMENTS, ItemEnchantments.EMPTY);
      }
-@@ -933,6 +_,8 @@
+@@ -933,12 +_,16 @@
      }
  
      public void forEachModifier(EquipmentSlot p_332001_, BiConsumer<Holder<Attribute>, AttributeModifier> p_330882_) {
++        // Neo: Use ItemStack extension method, which fires an event for mod-added attributes modifiers
 +        this.getAttributeModifiers(p_332001_).forEach(p_330882_);
-+        if (true) return;
++        if (false) {
          ItemAttributeModifiers itemattributemodifiers = this.getOrDefault(DataComponents.ATTRIBUTE_MODIFIERS, ItemAttributeModifiers.EMPTY);
```

Comment:
Shouldn't we just patch this to `getAttributeModifiers` instead?

Reply:
What do you mean? The patch replaces the disabled portion of code with `IItemStackExtension#getAttributeModifiers`, since that extension method fires the appropriate item attribute modifiers event.